Lib/test/test_format.py

```diff
@@ -519,5 +519,27 @@ def test_with_an_underscore_and_a_comma_in_format_specifier(self):
         with self.assertRaisesRegex(ValueError, error_msg):
             '{:_,}'.format(1)
 
+    @support.cpython_only
```

Comment:
Should this really be cpython only? Everywhere else in the file error messages are tested across all platforms.

Reply:
Removed.